KWZP2019/KWZP2019/MachinesForm.cs

```diff
@@ -35,11 +35,9 @@ private void btnReturnMain_Click(object sender, EventArgs e)
 
         void PopulateDataGridView()
         {
+            dgvMachine.DataSource = null;
```

Comment:
po co to skoro niżej przypisana wartość?

Reply:
Usunięte